webdriver-spec.html

```diff
@@ -1790,29 +1790,36 @@ <h3>Handling Errors</h3>
 <section>
 <h2>Protocol Extensions</h2>
 
-<p>The protocol is designed to allow extension to meet vendor-specific needs.
- Commands that are specific to a user agent
+<p>Using the terminology defined in this section, others may define additional
+ commands that seamlessly integrate with the standard protocol. This allows
+ vendors to expose functionality that is specific to their user agent, and it
+ also allows other web standards to define commands for automating new platform
+ features.
+
+<p>Commands defined in this way
  are called <dfn data-lt="extension command">extension commands</dfn>
  and behave no differently than other <a>commands</a>;
  each has a dedicated HTTP endpoint and a set of <a>remote end steps</a>.
 
 <p>Each <a>extension command</a> has an associated
- <dfn>extension command name</dfn>
+ <dfn>extension command URI Template</dfn>
  that is a <a>URI Template</a> string,
  and which should bear some resemblance to what the command performs.
- The name is used to form an <a>extension command</a>’s
- <a data-lt="extension command URI Template">URI Template</a>.
-
-<p>The <a>extension command</a>’s <dfn>extension command URI Template</dfn>
- is a <a>URI Template</a> composed of the <a>extension command prefix</a>,
- followed by "<code>/</code>",
- and the <a>extension command</a>’s <a data-lt="extension command name">name</a>.
- The <a>extension command URI Template</a>,
+ This value,
  along with the HTTP method and <a>extension command</a>,
  is added to the <a>table of endpoints</a>
  and thus follows the same rules for <a>request routing</a>
  as that of other built-in <a>commands</a>.
 
+<p>In order to avoid potential resource conflicts with other implementations,
+ vendor-specific <a>extension command URI Template</a>s SHOULD begin with one
```

Comment:
You can add `<dfn data-lt="extension command URI Templates">…</dfn>` for the plural form on :1805.

Reply:
It looks like the convention is to prefer the singular for term definitions, so I've done the inverse:

    <a data-lt="extension command URI Template">extension command URI Templates</a>

Does that seem okay to you?